src/item.cpp

```diff
@@ -1528,7 +1528,13 @@ void item::basic_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
         // Display any minimal stat or skill requirements for the item
         std::vector<std::string> req;
         if( get_min_str() > 0 ) {
-            req.push_back( string_format( "%s %d", _( "strength" ), get_min_str() ) );
+            avatar &viewer = get_avatar();
+            if( has_flag( flag_STR_DRAW ) && ranged::get_str_draw_penalty( *this, viewer ) < 1.0f ) {
+                req.push_back( string_format( "%s %d (damage/range penalty %.2f)", _( "strength" ), get_min_str(),
```

Comment:
```suggestion
                req.push_back( string_format( _( "%s %d (damage/range penalty %.2f)" ), _( "strength" ), get_min_str(),
```
The string contains translatable text, so it needs to be translated.